cosmo_tester/framework/testenv.py

```diff
@@ -103,11 +103,17 @@ def __init__(self):
         self._initial_cwd = os.getcwd()
         self._global_cleanup_context = None
         self._management_running = False
+        self._additional_management_running = []
         self.rest_client = None
+        self.additional_rest_clients = []
```

Comment:
To the reviewer:
We've thought about the following improvement - we could always have a list (even if we have one manager, client, management_ip etc) and a bunch of @property methods defined, like:

@property
def rest_client(self):
    return self.rest_clients[0]

for compatibility with existing tests. This way we could avoid all those "additional.." properties